### PR TITLE
release 0.5.0

### PR DESCRIPTION
**Pull request by wllenyj**
See CHANGELOG.md file for details.

Signed-off-by: wanglei01 <wllenyj@linux.alibaba.com>

### Summary of the PR

*Please summarize here why the changes in this PR are needed.*

### Requirements

Before submitting your PR, please make sure you addressed the following
requirements:

- [x] All commits in this PR are signed (with `git commit -s`), and the commit
  message has max 60 characters for the summary and max 75 characters for each
  description line.
- [x] All added/changed functionality has a corresponding unit/integration
  test.
- [x] Any newly added `unsafe` code is properly documented.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-## [Unreleased]
 
-### Added
+## [0.5.0]
 
 ### Changed
-
-### Fixed
-
-### Deprecated
```
Comment (sboeuf) [inline CHANGELOG.md:10]: Don't change the "Unreleased" section as it provides some sort of template.

Comment (wllenyj) [reply CHANGELOG.md]: thanks, fixed

**Review by sboeuf — APPROVED** [review]

**Comment by wllenyj** [comment]
Should #116 be included in this release?

**Comment by sboeuf** [comment]
> Should #116 be included in this release?

No it's not even merged yet (still under discussion).

**Comment by wllenyj** [comment]
> > Should #116 be included in this release?
> 
> No it's not even merged yet (still under discussion).

@sboeuf OK, thanks.

**Review by jiangliu — APPROVED** [review]

**Pull request merged**